docs/docs/users/reference/env_variables.md

```diff
@@ -28,6 +28,8 @@ process.
 | `FOREST_MAX_FILTER_HEIGHT_RANGE`                          | positive integer                 | 2880                                           | 2880                                                          | The maximum filter height range allowed, a conservative limit of one day         |
 | `FOREST_STATE_MIGRATION_THREADS`                          | integer                          | Depends on the machine.                        | 3                                                             | The number of threads for state migration thread-pool. Advanced users only.      |
 | `FOREST_CONFIG_PATH`                                      | string                           | /$FOREST_HOME/com.ChainSafe.Forest/config.toml | `/path/to/config.toml`                                        | Forest configuration path. Alternatively supplied via `--config` cli parameter.  |
+| `FOREST_TEST_RNG_FIXED_SEED`                              | non-negative integer             | empty                                          | 0                                                             | Override thread RNG with a reproducible one seeded by the value                  |
+| `FOREST_TEST_OS_RNG_FIXED_SEED`                           | non-negative integer             | empty                                          | 0                                                             | Override OS RNG with a reproducible one seeded by the value                      |
```

Comment:
Can this be a single variable?

Reply:
I used different variables because the 2 RNGs have different security implications in Forest. I can merge them if you prefer.

Reply:
They do, but once you start tinkering with `FOREST_TEST_` you should not expect any security. :rofl: